Sources/Tracking/Type/Aliases.swift

```diff
@@ -0,0 +1,12 @@
+import Common
+import Foundation
+
+/*
+ Contains public aliases to expose public structures from the 'Common' module
+ to the 'Tracking' module. The goal is that customers do not need to import
+ the 'Common' module in their code. But, some data structures need to exist
+ in 'Common' as they are used in the 'Common' code.
+ */
+
+public typealias Region = Common.Region
```

Comment:
This file was added [from this PR](https://github.com/customerio/customerio-ios/pull/166) so I could get Remote Habits to build successfully.